test/server/servers/SockJSServer.test.js

```diff
@@ -84,9 +84,20 @@ describe('SockJSServer', () => {
         done();
       }, 3000);
     });
+
+    it('should not throw an exception when connection is null', (done) => {
+      expect(() => {
+        socketServer.onConnection(() => {});
```

Comment:
Before this `expect` test, could you make a function `const cb = jest.fn()`, then pass it into the onConnection like: `onConnection(cb)`. Then after this confirm that the callback is actually being called with the expected values like: `expect(cb.mock.calls[0]).toEqual([null, null])`.